docs/connect.rst

```diff
@@ -0,0 +1,52 @@
+.. _schemaregistry_kafka_connect:
+
+Kafka Connect
+=============
+
+Kafka Connect and Schema Registry integrate to capture schema information from connectors. :ref:`Kafka Connect converters <connect_converters>`
+provide a mechanism for converting data from the internal data types used by Kafka Connect to data types represented as Avro.
+The AvroConverter will automatically register schemas generated by source connectors. Sink Connectors will receive schema
```

Comment:
One more? `Sink Connectors` -> `Sink connectors`? (btw I was fine with Sink and Source, only "connectors" was lower case across the rest of the doc). Sorry for the nits